looker_deployer/commands/deploy_content_export.py

```diff
@@ -0,0 +1,78 @@
+import os
+import subprocess
+import logging
+from pathlib import Path
+from looker_deployer.utils import deploy_logging
+from looker_deployer.commands.deploy_content import get_gzr_creds
+
+
+logger = deploy_logging.get_logger(__name__)
+
+
+def export_spaces(env, ini, path, debug=False):
+    host, port, client_id, client_secret, verify_ssl = get_gzr_creds(ini, env)
+
+    gzr_command = [
+        "gzr",
+        "space",
+        "export",
+        "1",
+        "--dir",
+        path,
+        "--host",
+        host,
+        "--port",
+        port,
+        "--client-id",
+        client_id,
+        "--client-secret",
+        client_secret
+    ]
+
+    # config parser returns a string - easier to parse that than convert to a bool
+    if verify_ssl == "False":
+        gzr_command.append("--no-verify-ssl")
+    if debug:
+        gzr_command.append("--debug")
+
+    # if we're running on windows we need to appropriately call the command-line arg"
+    if os.name == "nt":
```

Comment:
Is this value really nt across all recent versions? Doubtful. Expect to replace this with a list of possible values...

Reply:
Yes, it actually is.

https://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python

The general consensus seems to be this is the preferred way to go.